azure-devops/run_build.yml

```diff
@@ -0,0 +1,51 @@
+parameters:
+  targetPlatform: 'x64'
+
+jobs:
+- job: vs2019_hosted_${{ parameters.targetPlatform }}
+  pool:
+    #name: Hosted Windows 2019 with VS2019
+    name: STL
+
+  variables:
+    vcpkgLocation: '$(Build.SourcesDirectory)/vcpkg'
+    vcpkgResponseFile: $(Build.SourcesDirectory)/vcpkg_windows.txt
+  steps:
+    - checkout: self
```

Comment:
Should I check "don't sync sources" on this page since there's a checkout step here?

![image](https://user-images.githubusercontent.com/1544943/65568649-76137180-df0f-11e9-9df8-12a88f7a2d1b.png)

Reply:
no, the checkout step is needed to let submodules be pulled